modules/claims_api/app/models/claims_api/veteran.rb

```diff
@@ -17,6 +17,8 @@ class Veteran
        gender
        birls_file_number
        uuid
+       icn_with_aaid
+       search_token
```

Comment:
I initially tried to add these with
```ruby
delegate :icn_with_aaid, to: :mpi, allow_nil: true
delegate :search_token, to: :mpi, allow_nil: true
```
but I was still getting complaints that `icn_with_aaid=` and `search_token=` were undefined methods for `ClaimsApi::Veteran`. I'm not sure if I was doing it wrong trying this other way or if I'm missing something with Ruby.